Header header
uint8 powerUpType
uint8 powerUpPercentage